node/cmd/guardiand/node.go

```diff
@@ -358,7 +359,7 @@ func runNode(cmd *cobra.Command, args []string) {
 
 		go func() {
 			logger.Info("status server listening on [::]:6060")
-			logger.Error("status server crashed", zap.Error(http.ListenAndServe(*statusAddr, router)))
+			logger.Error("status server crashed", zap.Error(http.ListenAndServe(*statusAddr, router))) // SECURITY: If making changes, ensure that we always do `router := mux.NewRouter()` before this to avoid accidentally exposing pprof
```

Comment:
Please put the comment above this line, the line length is comically long otherwise.

Reply:
same.